crates/ruff_cli/src/printer.rs

```diff
@@ -476,41 +525,3 @@ impl FixableSummary {
         self.applicable > 0
     }
 }
-
-impl Display for FixableSummary {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
I don't think this belongs here, this is one specific message derived from this struct.